packages/ui5-application-writer/src/data/ui5Libs.ts

```diff
@@ -1,6 +1,14 @@
+import { defaultUI5Libs } from "./defaultSettings";
+
+/**
+ * Merges the specified ui5 libs with the defaults.
+ * 
+ * @param ui5Libs - The ui5 libraries to be merged with the defaults
+ * @returns UI5 libs with defaults
+ */
 export const getUI5Libs = (ui5Libs?: string | string[]): string[] => {
     const libs = Array.isArray(ui5Libs) ? ui5Libs : ui5Libs?.split(',') || [];
-    return ['sap.m'].concat(libs).filter((value, index, self) => {
+    return defaultUI5Libs.concat(libs).filter((value, index, self) => {
```

Comment:
Do we need to handle the case where the default libs are also passed in? We'll have duplicated entries then?

Reply:
That occurred to me also, but this filter condition removes duplicates.